### PR TITLE
Add role enum to frontend

### LINKED ISSUE #372: Replace hard coded Role ints across FE app
## Description

This is to prevent serious bugs. If it's not done already we should define an Enum in one place to extract the correct value for a Role. It doesn't matter if this is in a test file or application code. Replace all hard coded ints with a call to retrieve the correct value from a single source. 

@jls-github identified two places where we are using hard coded ints. 🙇‍♂️ 

## Acceptance criteria

- Replace hardcoded ints in `dashboard.js` currently at line # 35 and 42 with a call to the FE enum.
- Remove any other hard coded ints that you can find.

### PR DESCRIPTION
**Pull request by jls-github**
### What issue is this solving?
Closes https://github.com/codeforpdx/dwellingly-app/issues/372

### Any helpful knowledge/context for the reviewer?
Is a `npm install` necessary? No
Any special requirements to test? No

### Please make sure you've attempted to meet the following coding standards
- [x] Code builds successfully
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code
- [x] There aren't any unnecessary commits or files changed (shouldn't touch the /stashed folder unless the issue requests it)

If you're having trouble meeting this criteria, feel free to reach out on Slack for help!

### CODE REVIEW

File: src/RoleEnum.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const RoleEnum = {
```
Comment (Chris-Boe) [inline src/RoleEnum.js]: Let's put this in a new folder under /src called /Enums so we can start a collection!

**Review by Chris-Boe — CHANGES_REQUESTED** [review]
Nice! I think we'll just want to move the file into a folder so we improve our organization of the app

**Comment by Chris-Boe** [comment]
Now that the fetch-join-staff PR was merged, let's also update that file to use the enum

File: src/RoleEnum.js (hunk 1 above)
Comment (jls-github) [reply src/RoleEnum.js]: Good idea. Just updated the file structure.

**Comment by jls-github** [comment]
> Now that the fetch-join-staff PR was merged, let's also update that file to use the enum

Just submitted a new commit with that implemented.

**Review by Chris-Boe — APPROVED** [review]

**Pull request merged**